projects/metis/src/app/dataset/depublication/depublication.component.ts

```diff
@@ -57,7 +57,10 @@ export class DepublicationComponent extends DataPollingComponent {
 
   formFile = this.fb.group({
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    depublicationFile: [(undefined as any) as File, [Validators.required, this.validateFileExtension]]
+    depublicationFile: [
+      (undefined as any) as File,
+      [Validators.required, this.validateFileExtension]
+    ]
   });
 
```

Comment:
I see here that format changed in another project, should this be part of a metis ticket? I see the ticket is for sandbox

Reply:
After discussing this is file was coincidentally updated as part of the prettier process.